packages/roosterjs-editor-dom/lib/utils/extractClipboardEvent.ts

```diff
@@ -104,3 +137,11 @@ function workaroundForEdge(html: string) {
 
     return html;
 }
+
+function getAsString(item: DataTransferItem): Promise<string> {
+    return new Promise<string>(resolve => {

```

Comment:
Do you have a preference for using promises over the async/await syntax?

Reply:
As long as promise can be used easily, I prefer promise. Because async/await will introduce extra bytes when compile to ES5.